### DISCUSSION TITLE
Logs are not getting captured from console if we use third party library for logging

### DISCUSSION THREAD

**Discussion by hraj6056** [grafana/faro-web-sdk#559] [General]
In my project i am using [loglevel](https://www.npmjs.com/package/loglevel) as console logger but console logs are not getting captured i have this initialize code`export const coreInit = (): Faro => {
	return initializeFaro({
		transports: [
			new FetchTransport({
				url: config.faro.url,
				apiKey: config.faro.apiKey,
				bufferSize: 1000,
				concurrency: 5,
				defaultRateLimitBackoffMs: 1000,
			}),
		],
		app: {
			name: 'Frontend',
			version: VERSION,
			environment: env.getEnvironment(),
		},
		instrumentations: [
			...getWebInstrumentations({
				captureConsole: true,
				captureConsoleDisabledLevels: [],
			}),
			new TracingInstrumentation(),
		],
		metas: [pageMeta, browserMeta],
		beforeSend: (item: TransportItem) => {
			delete item.meta.sdk;
			return item;
		},
	});
};`

**Comment by codecapitano** [comment]
Hi @hraj6056 As far as I can see `loglevel` monkey patches the console. 
Faro does so as well. 
I have the suspicion that loglevel is initialized after Faro thus overwriting the Faro console.* patches.

**Reply by hraj6056** [reply]
Thanks for the help, loglevel was overwriting the faro console, fixed it.

**Reply by codecapitano** [reply]
Thanks again for reporting it.
It's a thing we should have a closer look at and see what we can do to prevent the issue.
